medusa/server/web/home/handler.py

```diff
@@ -1652,9 +1660,21 @@ def editShow(self, show=None, location=None, allowed_qualities=None, preferred_q
                 logger.log("Unable to force an update on scene numbering for show '{show}': {error}".format
                            (show=show_obj.name, error=e.message), logger.WARNING)
 
-            # Must erase cached results when toggling scene numbering
+            # Must erase cached DB results when toggling scene numbering
             self.erase_cache(show_obj)
 
+            # Erase parsed cached names as we are changing scene numbering
+            show_obj.flush_episodes()
```

Comment:
Because user change the show format. So all loaded show needs to be updated again.